tools/autodeployment/server.js

```diff
@@ -19,10 +19,24 @@ handler.on('error', (err) => {
   console.error('Error:', err.message);
 });
 
-handler.on('push', (event) => {
-  const { name } = event.payload.repository;
+if (DEPLOY_TYPE === 'production') {
+  handler.on('release', (event) => {
+    const { name } = event.payload.repository;
 
-  if (name === REPO_NAME) {
-    shell.exec('./deploy_stage.sh');
-  }
-});
+    if (name === REPO_NAME) {
+      // Execute the shell script and pass two arguments, our deployment type and the .env file to use
+      shell.exec(`./deploy.sh ${DEPLOY_TYPE}`);
```

Comment:
I would use the string `production` here vs. `${DEPLOY_TYPE}` since you know it's `production` from line 22.

Reply:
changed to use string